src/components/Calendar/src/Calendar.vue

```diff
@@ -121,214 +109,180 @@ export default {
 
 	props: {
 		/**
-		 * Language specific format to use
-		 */
-		locale: {
-			type: String,
-			default: 'en-US',
-		},
-		/**
-		 * Selected date value (YYYY-MM-DD)
+		 * Selected date value in ISO format.
 		 */
 		selectedDate: {
 			type: String,
-			default: '',
-			validator: isDateFormat,
+			default: undefined,
+			validator: isIsoFormat,
 		},
 		/**
-		 * Disable the dates before the min-date
+		 * Disable the dates before the min-date in ISO format.
 		 */
 		minDate: {
 			type: String,
-			default: '',
-			validator: isDateFormat,
+			default: undefined,
+			validator: isIsoFormat,
 		},
 		/**
-		 * Disable the dates after the max-date
+		 * Disable the dates after the max-date in ISO format.
 		 */
 		maxDate: {
 			type: String,
-			default: '',
-			validator: isDateFormat,
+			default: undefined,
+			validator: isIsoFormat,
 		},
 		/**
-		 * List of disabled dates
+		 * List of disabled dates in ISO format.
 		 */
 		disabledDates: {
 			type: Array,
 			default: () => ([]),
-			validator: (v) => v.every((date) => isDateFormat(date)),
+			validator: (disabledDates) => disabledDates.every((date) => isIsoFormat(date)),
+		},
+		/**
+		 * Calendar locale. Defaults to browser locale.
+		 */
+		locale: {
+			type: String,
+			default: undefined,
```

Comment:
`toLocale` methods detect the browser's locale when set to `undefined`. This should be the default behavior instead of `en-US`.